pages/index.tsx

```diff
@@ -98,6 +99,18 @@ const Icebreaker: NextPage<Props> = ({
           handleCopyUrlClick={handleCopyUrlClick}
         />
       </Card>
+      <div
+        className="group mx-4 mt-auto mb-4 inline-flex flex-col items-center justify-center gap-2 text-center text-white"
+      >
+          <a
```

Comment:
-1 instead of `a` let's use `Button` component, we're not doing anything with URL here so there's no need for `a`

Reply:
<img width="728" alt="Captura de Pantalla 2022-08-17 a la(s) 4 51 18 p m" src="https://user-images.githubusercontent.com/3276087/185249658-c655117a-f4aa-44dd-bd5c-11e6b804f5bc.png">

If I use `Button` then I get some unwanted button styles, like for `hover` and `focus` states. Is it OK if I somehow override these?